server/mentionscrawler/blueprints/mention.py

```diff
@@ -41,3 +43,24 @@ def mention_response(user):
                          }
         output_mentions.append(output_mention)
     return jsonify(output_mentions), 200
+
+# Get details of a single mention
+@mention_bp.route("/mention", methods=["POST"])
```

Comment:
if we're going to go this route this should be a get request, and then you'd just use a parameter in mentions, if you'd like I can implement that for you

Reply:
or if you'd prefer to do it yourself, check out how I did the site route (but as a get instead of a post)

Reply:
Sure go ahead